src/Display/Slider/Slider.tsx

```diff
@@ -148,7 +148,7 @@ class Slider extends React.Component<Props, State> {
     window.addEventListener('resize', this.setSize);
   }
 
-  componentWillReceiveProps(nextProps: Props) {
+  UNSAFE_componentWillReceiveProps(nextProps: Props) {
```

Comment:
somehow we missed this intended change haha https://github.com/glints-dev/glints-aries/pull/269#discussion_r404549173